source/tags/Heading/README.md

```diff
@@ -1,7 +1,17 @@
 # Heading
 
-## API
+Headings replace generic h1, h2, h3, h4, h5, h6
```

Comment:
I'm not convinced that _replace_ and _generic_ are the right words here. _Replace_ feels wrong given that the tag does return a HTML tag and _generic_ would maybe be better as _default_. I'm listening to a story about a drowned mouse, so I'm unable to write it myself at the moment, but we need new words here.